src/main/java/core/basesyntax/service/ReportWriterToFileImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.DataBase;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportWriterToFileImpl implements ReportWriterToFile {
+    @Override
+    public void createReport(String filePath, ReportCreator reportCreator) {
```

Comment:
almost :) in your case, IO method still can't work without business service, but it should
```suggestion
    public void createReport(String filePath, String content) {
```